src/components/upload-panel/upload-panel.tsx

```diff
@@ -83,6 +81,8 @@ export const UploadPanel = ({
   onFileUploaded,
   onFileEvent,
 }: UploadProps) => {
+  const UPLOAD_INPUT_ID = useMemo(() => `upload-file-input${crypto.randomUUID()}`, []);
```

Comment:
What does this fix? It would be good to have a list of changes in the PR description with screenshots when applicable

Reply:
If you have several upload panels open at the same time (for example in comparison mode) all actions go to the first one and you're unable to choose a file on the second one. This happens because all file inputs and divs that catch the actions have the same id. This fix generates the unique id for every upload panel so actions will be passed inside every upload panel separately

Reply:
The stab needs to be there because `jsdom` doesn't have a stub for browser's `crypto`. And since we don't have tests where two upload panels would be shown at the same time, so `randomUUID: () => ""` is enough for all our tests.

Reply:
Can we avoid hacks?
The offered approach is very weird:
* We make a global variable that is bad practice;
* We declare `randomUUID` that doesn't return a UUID;
* Why `crypto.reandomUUID`? Is there any reason to mock nodejs API?

For example, we can use Redux to generate unique application-wide integer ids.